Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarCloudWebServiceTest.cs

```diff
@@ -138,5 +153,140 @@ public async Task IsServerLicenseValid_AlwaysTrue()
             isValid.Should().BeTrue();
             logger.AssertDebugMessageExists("SonarCloud detected, skipping license check.");
         }
+
+        #region CACHESTUFF
```

Comment:
We are not using region. We can remove this.

Reply:
Sometimes, if there are lots of tests specific to a method, we create a partial class in a separate file and add the related tests there.

Reply:
It was mostly to organize it while i was working on it, but I forgot to remove it, thanks!